packages/protocol/migrations_sol/constants.sol

```diff
@@ -7,7 +7,8 @@ contract MigrationsConstants is TestConstants {
   address constant DEPLOYER_ACCOUNT = 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266;
 
   // List of contracts that are expected to be in Registry.sol
-  string[27] contractsInRegistry = [
+  // TODO: Probably should be synced with Migration.s.sol
+  string[26] contractsInRegistry = [
```

Comment:
I couldn't figure out how to make the compiler come up with the number of items automatically, maybe we should move to json at some point